webapp/channels/src/components/sidebar_right_menu/sidebar_right_menu.tsx

```diff
@@ -26,52 +24,53 @@ type Props = {
 
 const ANIMATION_DURATION = 500;
 
-export default class SidebarRightMenu extends React.PureComponent<Props> {
-    handleEmitUserLoggedOutEvent = () => {
-        GlobalActions.emitUserLoggedOutEvent();
-    };
-
-    render() {
-        let siteName = '';
-        if (this.props.siteName != null) {
-            siteName = this.props.siteName;
-        }
-        let teamDisplayName = siteName;
-        if (this.props.teamDisplayName) {
-            teamDisplayName = this.props.teamDisplayName;
-        }
+const SidebarRightMenu = ({
+    siteName: defaultSiteName,
+    teamDisplayName: defaultTeamDisplayName,
+    isOpen,
+    isMobileView,
+}: Props) => {
```

Comment:
I don't see actions in the list of props. If it is not used, can you remove it also from the props type, tests files and index file if needed?